config/v1/types_tlssecurityprofile.go

```diff
@@ -107,6 +110,8 @@ type TLSSecurityProfile struct {
 	//     - ECDHE-ECDSA-AES128-GCM-SHA256
 	//   minTLSVersion: TLSv1.1
 	//
+	// NOTE: Currently unsupported.
```

Comment:
unsupported or rejected?

Reply:
I think unsupported is the better wording since `IngressControllerSpec` will not reject it AFAIK, but does not support it (I believe they just fall back to Intermediate). We would reject it in the APIServer configuration, and the reason is the same with ingress - the components that would react to such configuration cannot process TLSv1.3.